crates/syntax/src/ast/token_ext.rs

```diff
@@ -445,6 +449,12 @@ mod tests {
         check_string_value(r"\foobar", None);
         check_string_value(r"\nfoobar", "\nfoobar");
         check_string_value(r"C:\\Windows\\System32\\", "C:\\Windows\\System32\\");
+        check_string_value(r"\x61bcde", "a\x62cde");
```

Comment:
Shouldn't the second string here be `abcde`?

Reply:
Thank you very much for reviewing. As you suggest, `abcde` would be clearer in intent, so I will modify the code accordingly. Also, since the existing test code is only for strings, I will add test code for byte strings.